common/changes/@microsoft/rush/jest-sharding_2024-04-26-17-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Adds support for sharding operations during task execution.",
```

Comment:
Does this require cobuilds?

Reply:
It shouldn't, there's nothing in the implementation that requires cobuilds to be set up.